### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by colin99d**
Added some tests.

### CODE REVIEW

**Comment by DidierRLopes** [comment]
I think this may be failing due to your BeautifulSoup version

File: tests/test_cryptocurrency/test_dd_controller.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    @check_print(assert_in="Moving back to")
+    def test_q(self):
+        self.cont.call_q(None)
+
+    def test_quit(self):
+        self.assertTrue(self.cont.call_quit(None))
+
+    @vcr.use_cassette(
+        "tests/cassettes/test_cryptocurrency/test_dd_controller.yaml",
+        record_mode="new_episodes",
```
Comment (aia) [inline tests/test_cryptocurrency/test_dd_controller.py:27]: record_mode="new_episodes" will run external HTTP requests on every run.
The test suit will be flaky as we will be load testing all of our API endpoints.

**Comment by DidierRLopes** [comment]
You can touch base with @Chavithra, he's working on the architecture to follow when developing tests.

**Review by Chavithra — COMMENTED** [review]
1/ Works fine, I can generate the `xls` (tested manually) from this method :
- function : CreateExcelFA::create_dcf
- file : gamestonk_terminal/stocks/fundamental_analysis/dcf_view.py
- command : (✨) (stocks)>(fa)> dcf

2/ These tests `passed` :

    tests/test_cryptocurrency/test_dd_controller.py::TestDDController::test_help PASSED
    tests/test_cryptocurrency/test_dd_controller.py::TestDDController::test_q PASSED
    tests/test_cryptocurrency/test_dd_controller.py::TestDDController::test_quit PASSED

3/ These test `failed` :

    tests/test_cryptocurrency/test_dd_controller.py::TestDDController::test_active FAILED

Seems to be because of this cassette record body :

    tests/cassettes/test_cryptocurrency/test_dd_controller.yaml

4/ This cassette seems to work fine with the test which uses it :

    tests/cassettes/test_fa/test_fa_business/test_management.yaml

  
5/ This cassette seems to be unused :

    tests/cassettes/test_fa/dcf_view.yaml

**Review by Chavithra — APPROVED** [review]
All the tests passes.

No idea why `test_active` failed last time I have tried.

**Pull request merged**